math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1256,7 +1256,7 @@ public:
       if (n != this->size()) {
          std::string msg = "Cannot index RVecN of size " + std::to_string(this->size()) +
                            " with condition vector of different size (" + std::to_string(n) + ").";
-         throw std::runtime_error(std::move(msg));
+         throw std::runtime_error(msg);
```

Comment:
Why is that an enhancement? (I.e. what was the clang-tidy message?)

Reply:
the ctor takes a `const std::string&`, so the `std::move` is doing nothing.